ui.js

```diff
@@ -25,11 +25,15 @@ var stopGame = function () {
 
 
 $(document).ready(function() {
-	//Must bind contextmenu event handler to each cell
-	//document.oncontextmenu = function() {return false;};
-	var grid = new Grid(5,5,2);
-	var report = grid.print();
-	
+	var grid;
+	var report;
+	$('#gridSetup').submit(function(){
+		var $values = $('#gridSetup').serializeArray();
+		grid = new Grid(parseInt($values[0].value),parseInt($values[1].value),parseInt($values[2].value));
+		// var grid = new Grid(7,7,5);
+		report = grid.print();
+	}); // <<<The program kicks me out here.>>>
```

Comment:
It's hard to know what is happening if you're not familiar with `<form>`. You made a small beginner's mistake by not preventing the default behavior of the form. Whenever you submit a form, it will make a synchronous `POST` request and refresh the page on the response. So you'll need to do this to stop the page refresh (which you were perceiving as the game kicking you out):
```javascript
$('#gridSetup').submit(function(e){ // the first and only parameter to this event handler is a JQuery event object
  e.preventDefault();
  var $values = $('#gridSetup').serializeArray();
  ...
```